source/incident-collaboration/overview.rst

```diff
@@ -1,23 +1,22 @@
 Overview
 ========
 
-Mattermost Incident Collaboration equips your team to coordinate, manage, and resolve incidents from within Mattermost using playbooks. Each playbook represents a recurring outcome that your teams collaborate to achieve, such as Service Outage Recovery or Customer Onboarding. For the participants, it prescribes processes such as task checklists, status updates, and retrospective reports. For the integrated tools, it configures the triggers to perform automate actions. And for stakeholders, it provides a single pane of glass for visibility into each run as well as aggregate insights over time.
+Mattermost Playbooks are a collaboration tool for teams. Teams run a playbook every time they want to orchestrate people, tools, and data to achieve that outcome as quickly as possible while providing visibility to stakeholders. Playbooks also allow teams to incorporate learnings from the retrospective to tweak and improve the playbook with every iteration.
```

Comment:
Not yet renamed, but I think this should read, `Mattermost Playbooks is a...` based on my understanding of the terminology. 

It gets tricky with `Playbooks also allow...` since that could either be a capitalized `playbooks` or an abbreviated `Mattermost Playbooks`. Propose we plan to write `Mattermost Playbooks` unconditionally when referring to the product, which then makes this case already correct.

Reply:
This also refers to `that outcome` but the definition of the specific outcome is only in the next paragraph, when it says `Each playbook represents a recurring outcome...`. Should we reorder this?